app/src/main/java/org/resolvetosavelives/red/patient/PatientWithAddressAndPhone.kt

```diff
@@ -6,7 +6,7 @@ import org.threeten.bp.Instant
 import org.threeten.bp.LocalDate
 import java.util.UUID
 
-@Deprecated(message = "You should use PatientSearchResult instead.")
+@Deprecated(message = "You should use PatientSearchResult instead.", level = DeprecationLevel.ERROR)
 data class PatientWithAddressAndPhone(
```

Comment:
Will compilation fail if I use this class? In that case, can we remove this class?

Reply:
Yes, removed. This was an oversight.